gnome-initial-setup/pages/language/cc-language-chooser.c

```diff
@@ -577,6 +577,7 @@ static void
 cc_language_chooser_finalize (GObject *object)
 {
 	CcLanguageChooser *chooser = CC_LANGUAGE_CHOOSER (object);
+    show_more (chooser);
```

Comment:
I would add a comment as to why this is needed, since it's not intuitive why showing more languages when closing the page would be useful.